library/common/extensions/key_value/platform/config.cc

```diff
@@ -4,10 +4,37 @@
 #include "envoy/config/common/key_value/v3/config.pb.validate.h"
 #include "envoy/registry/registry.h"
 
+#include "library/common/api/external.h"
+#include "library/common/data/utility.h"
+#include "library/common/extensions/key_value/platform/c_types.h"
+
 namespace Envoy {
 namespace Extensions {
 namespace KeyValue {
 
+class PlatformInterfaceImpl : PlatformInterface, public Logger::Loggable<Logger::Id::filter> {
+public:
+  PlatformInterfaceImpl(const std::string& name)
+      : bridged_store_(*static_cast<envoy_kv_store*>(Api::External::retrieveApi(name))) {}
+
+  ~PlatformInterfaceImpl() override {}
+
+  std::string read(const std::string& key) const override {
+    envoy_data bridged_key = Data::Utility::copyToBridgeData(key);
+    envoy_data bridged_value = bridged_store_.read(bridged_key, bridged_store_.context);
+    return Data::Utility::copyToString(bridged_value);
+  }
+
+  void save(const std::string& key, const std::string& contents) override {
+    envoy_data bridged_key = Data::Utility::copyToBridgeData(key);
+    envoy_data bridged_value = Data::Utility::copyToBridgeData(contents);
+    bridged_store_.save(bridged_key, bridged_value, bridged_store_.context);
+  }
+
```

Comment:
I would have expected a remove() method. Is that not required?

Reply:
@alyssawilk shared that you all didn't have a use case for it right now (which I sort of found surprising, but then again, if you supply the KV store implementation, you can always remove stuff from it elsewhere). I went ahead and wired `remove` at the other layers anyways, simply because I do expect we will have a use for it.